### PR TITLE
mini update 13: update definitions

### PR DESCRIPTION
**Pull request by pmengelbert**
This PR is based on #175 .

See the relevant diff below:
https://github.com/bloodorangeio/distribution-spec/compare/mini-update-12...bloodorangeio:mini-update-13?expand=1

### CODE REVIEW

File: spec.md [hunk 1]
```diff
@@ -285,7 +285,47 @@ An attempt to pull a nonexistent repository MUST return response code `404 Not F
...
 #### Content Discovery
 
-TODO: describe the Content Discovery category and the high-level details
+Content Discovery provides a way of discovering which tags are available for a given repository.
+
+To fetch the list of tags, perform a `GET` request to a path in the following format:
+`/v2/<name>/tags/list`
+
+`<name>` is the namespace of the repository. Assuming a repository is found, Content Discovery endpoints MUST return a
+`200 OK` response code. The list of tags MAY be empty, if there are no tags on the repository. If the list is not empty,
```
Comment (hallyn) [inline spec.md:295]: nit - maybe drop the comma after 'empty'?

File: spec.md [hunk 2]
```diff
@@ -285,7 +285,47 @@ An attempt to pull a nonexistent repository MUST return response code `404 Not F
...
+In this case, the path will look like the following:
+`/v2/<name>/tags/list?n=<int>`
+
+`<name>` is the namespace of the repository, and `<int>` is an integer specifying the number of tags requested. The response
+to such a request MAY return fewer than `<int>` results, but ONLY when the total number of tags attached to the repository
+is less than `<int>`. Otherwise, the response MUST include `<int>` results. Without the `last` query parameter (described
+next), the list returned will start at the beginning of the list and include `<int>` results. As above, the tags MUST be
+in lexical order.
+
+The `last` query parameter provides further means for limiting the number of tags. It is used exclusively in combination with the
```
Comment (hallyn) [inline spec.md:322]: last seems weird.  I know startafter is longer, but it's more correct, no?

**Comment by hallyn** [comment]
(weird, gh was saying there were only 2 comments when i hit approve, so i thought maybe it was for just that commit's review.  Would have marked it 'changes requested' otherwise)

File: spec.md [hunk 3]
```diff
@@ -86,7 +86,29 @@ Registry providers can self-cetify by submitting conformance results to [opencon
 
 #### Pull
 
-TODO: describe the Pull category and the high-level details
+##### Pulling Blobs
+
+To pull a blob, perform a `GET` request to a url in the following form:
+`/v2/<name>/blobs/<digest>`
+
+`<name>` is the namespace of the repository, and `<digest>` being the blob's digest.
```
[the remark this answers is not in the document]
Comment (pmengelbert) [reply spec.md]: got this in the underlying PR, thanks

**Comment by jdolitsky** [comment]
Closing this in favor of https://github.com/opencontainers/distribution-spec/pull/178

@hallyn if you could port your comments there - thank you!

**Pull request closed without merging**